src/metabase/lib/schema/aggregation.cljc

```diff
@@ -0,0 +1,20 @@
+(ns metabase.lib.schema.aggregation
+  (:require
+   [metabase.lib.schema.common :as common]
+   [metabase.lib.schema.expression :as expression]
+   [metabase.util.malli.registry :as mr]))
+
+(mr/def ::sum
+  [:tuple
+   [:= :sum]
+   ::common/options
+   [:ref ::expression/number]])
```

Comment:
Added a schema for a `:sum` aggregation just as a placeholder so I could test aggregations